pkg/kubernetes/scripts/app.js

```diff
@@ -210,8 +209,10 @@
 
             var promise = discoverSettings().then(function(settings) {
                 var ret = [];
-                if (settings.flavor === "openshift")
+                if (settings.flavor === "openshift") {
+                    ret.push(loader.watch("projects", $rootScope));
```

Comment:
OK. It felt a bit like an opportunistic thing that we neither rely on nor test, but it seems that's pretty much exactly its intention :-) At least I don't see how it could make matters worse.